api_client/python/timesketch_api_client/client.py

```diff
@@ -493,3 +495,78 @@ def refresh_oauth_token(self):
             return
         request = google.auth.transport.requests.Request()
         self.credentials.credential.refresh(request)
+
+
+    def list_sigma_rules(self, as_pandas=False):
+        """Get a dict of all sigma rules that the user has access to.
+        
+        Args:
+            as_pandas: Boolean indicating that the results will be returned
+                as a Pandas DataFrame instead of a list of dicts.
+
+        Returns:
+            A list with dict objects with the information about sigma rule,
+            unless as_pandas is set, then the function returns a DataFrame
+            object.
+        """
+
+        sigma_rules = []
+        response = self.fetch_resource_data('sigma/')
+
+        if not as_pandas:
+            return response
+
+        recs = response['objects']
+        rules_pandas = pandas.DataFrame.from_records(recs)
+
+        return rules_pandas
+
+       
+
+    def get_sigma_rule(self, rule_uuid):
+        """Get a sigma rule.
+
+        Args:
+            rule_uuid: UUID of the Sigma rule.
+
+        Returns:
+            Instance of a Sigma object.
+        """
+        return sigma.Sigma(rule_uuid, api=self)
+
+
+    def get_sigma_rule_as_json(self, rule_uuid, as_pandas=False):
```

Comment:
@berggren what is the better way for frontend and others, return a json or make it an object (where the content is mostly really better suited in a json)

Reply:
I would absolutely not do this... rather just use the `get_sigma_rule`, that returns back a Sigma object... which could easily have `to_json` function

Reply:
yes this can be removed, if someone uses .data() should be good enough, removed it.